docs/Reference/CLI/CLI-Syntax.md

```diff
@@ -89,6 +89,72 @@ Timeout period (in milliseconds) for downstream requests. Default is 5000.
 --downstream-http-request-timeout=3000
 ```
 
+### downstream-http-tls-enabled
+
+Enable or disable [TLS for server connections](../../Concepts/TLS.md).
+
+```bash tab="Syntax"
+--downstream-http-tls-enabled
+```
+
+```bash tab="Example"
+--downstream-http-tls-enabled
+```
+
+### downstream-http-tls-ca-auth-enabled
+
+Allow connections to servers with trusted CAs.
+
+Defaults to `true`.
+
+```bash tab="Syntax"
+--downstream-http-tls-ca-auth-enabled[=<true|false>]
+```
+
+```bash tab="Example"
+--downstream-http-tls-enabled=false
+```
+
+### downstream-http-tls-keystore-file
+
+Keystore file that is presented to the server during authentication.
```

Comment:
It should be rephrased something similar to 

> Keystore file (in PKCS #12 format) that contains private key and the certificate presented to the server during authentication.

Reply:
Done.